terraform/stacks/external/outputs.tf

```diff
@@ -3,10 +3,10 @@ output "cdn_broker_username" {
   value = "${module.cdn_broker.username}"
 }
 output "cdn_broker_access_key_id_prev" {
-  value = "${module.cdn_broker.access_key_id_prev}"
+  value = ""
```

Comment:
That's the idea--we should only have to blank out each credential pair in one place.